joomla/joomla-cms
# [staging] Adding missing lang strings for XTD plugins

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/25344
Replaces https://github.com/joomla/joomla-cms/pull/25472
### Summary of Changes
Loading main administrator joomla xx-XX.ini is not a good idea as it may override some frontend xx-XX.ini strings.
The only reason `$lang->load('joomla', JPATH_ADMINISTRATOR);` was added for com_menus and com_modules was to get these strings from backend.
These calls are now deleted.

The correct solution was to add then in frontend.
Some looks weird to add:
```
JTOGGLE_HIDE_SIDEBAR="Hide the sidebar"
JTOGGLE_SHOW_SIDEBAR="Show the sidebar"
```
but as they show as untranslated when using the Field xtd, I added them.
_(it would be interesting to find out why they are needed there and if corrected, delete them in 4.0)_

### Testing Instructions
Set Language debug on and load each xtd plugin when editing an article in frontend.
Check if any string is missing.

### After patch
The main display change with Protostar is the corrected `JGLOBAL_LIST_ALIAS="(<span>Alias</span>: %s)"` string.

@SharkyKZ @Quy
